File: omnes.Web/Modules/Parametros/AreasTrabajo/AreasTrabajoForm.cs
using Serenity.ComponentModel;

namespace omnes.Parametros.Forms;

[FormScript("Parametros.AreasTrabajo")]
[BasedOnRow(typeof(AreasTrabajoRow), CheckNames = true)]
public class AreasTrabajoForm
{
    public int IdTennant { get; set; }
    public int IdEmpresa { get; set; }
    public string Descripcion { get; set; }
    public bool Baja { get; set; }
}

File: omnes.Web/Modules/ServerTypes/Parametros/AreasTrabajoForm.ts
import { IntegerEditor, StringEditor, BooleanEditor, PrefixedContext } from "@serenity-is/corelib";
import { initFormType } from "@serenity-is/corelib/q";

export interface AreasTrabajoForm {
    IdTennant: IntegerEditor;
    IdEmpresa: IntegerEditor;
    Descripcion: StringEditor;
    Baja: BooleanEditor;
}

export class AreasTrabajoForm extends PrefixedContext {
    static formKey = 'Parametros.AreasTrabajo';
    private static init: boolean;

    constructor(prefix: string) {
        super(prefix);

        if (!AreasTrabajoForm.init)  {
            AreasTrabajoForm.init = true;

            var w0 = IntegerEditor;
            var w1 = StringEditor;
            var w2 = BooleanEditor;

            initFormType(AreasTrabajoForm, [
                'IdTennant', w0,
                'IdEmpresa', w0,
                'Descripcion', w1,
                'Baja', w2
            ]);
        }
    }
}